deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -38,7 +38,7 @@ spec:
   subresources: {}
   validation:
     openAPIV3Schema:
-      description: KogitoApp is a project prescription running a Kogito Runtime Service.
+      description: KogitoApp is a project prescription running a Kogito service.
```

Comment:
@ricardozanini Some descriptions end with a period, some don't. They all should, for consistency, but honestly up to you how particular you want to be, given its purpose.

Reply:
Updated!